PowerShell/JumpCloud Module/Private/Policies/Get-JCPolicyTemplateConfigField.ps1

```diff
@@ -0,0 +1,84 @@
+function Get-JCPolicyTemplateConfigField {
+    # TemplateID of the policy to be modified
+    [CmdletBinding()]
+    param (
+        [Parameter(mandatory = $true)]
+        [System.String]
+        $templateID
+    )
+    begin {
+        # TODO: validate templateID
+        # $template = Get-JcSdkPolicyTemplate -Id $templateID
+        $headers = @{}
+        $headers.Add("x-api-key", $env:JCApiKey)
+        $headers.Add("x-org-id", $env:JCOrgId)
+        $template = Invoke-RestMethod -Uri "https://console.jumpcloud.com/api/v2/policytemplates/$templateID" -Method GET -Headers $headers
+        # TODO: set this globally
+        $configMapping = @{
+            checkbox       = 'boolean'
+            singlelistbox  = 'listbox'
+            table          = 'table'
+            customRegTable = 'table'
+            textarea       = 'string'
+            text           = 'string'
+            file           = 'file'
+            select         = 'multi'
+            number         = 'int'
+        }
+    }
+    process {
+        # build object for templateID with validation of types
+        $objectMap = New-Object System.Collections.ArrayList
+        foreach ($field in $template.ConfigFields) {
+            # Write-Host "$($field.Name) accepts $($configMapping[$field.DisplayType]) input"
+            # if ($field.DisplayOptions.Values.values)
+            if ($Field.DisplayType -eq 'select') {
+                $ValidationObject = @()
+                $validationRows = $field.DisplayOptions.select.Count
+                for ($i = 0; $i -lt $validationRows; $i++) {
+                    # Get Both Values:
+                    $val1 = $field.DisplayOptions.select[$i].text
+                    $val2 = $field.DisplayOptions.select[$i].value
+                    # Determine which one is an int
+                    # TODO: Joe to review here is this necessary for non-sdk calls?
```

Comment:
Might need to clean up some of these comments/todos?

Reply:
removed